ocaml/xapi/quicktest_cbt.ml

```diff
@@ -167,10 +169,11 @@ let vdi_clone_copy_test ~session_id ~sR ~vDI =
     [ true , vDI , "VDI.copy erroneously reset the original VDI's cbt_enabled to false"
     ; false , into_vdi , "VDI.copy failed to initialise cbt_enabled to false"
     ; false , vdi_copy_fresh , "VDI.copy erroneously reset the copied VDI's cbt_enabled field to true"
+    ; false , vdi_clone , "VDI.copy erroneously reset the cloned VDI's cbt_enabled field to true"
```

Comment:
I think this text should say `"VDI.clone failed to set cbt_enabled to false"` because we're doing the same check here as on line 156 - we don't call `VDI.copy` on `vdi_clone`, if I'm not mistaken, so it cannot reset the cloned VDI's fields?

Reply:
So I think this means that we can remove the `assert_cbt_status` check for `vdi_clone` at line 156, because we're doing the same check here?